frontend/src/lib/ocp-utils.ts

```diff
@@ -8,22 +8,12 @@ function api<T>(url: string, headers?: Record<string, unknown>): Promise<T> {
     })
 }
 
-export function launchToOCP(urlSuffix: string, newTab: boolean, onError?: VoidFunction) {
-    api<{ data: { consoleURL: string } }>(
-        '/multicloud/api/v1/namespaces/openshift-config-managed/configmaps/console-public/'
-    )
-        .then(({ data }) => {
-            if (newTab) {
-                window.open(`${data.consoleURL}/${urlSuffix}`)
-            } else {
-                location.href = `${data.consoleURL}/${urlSuffix}`
-            }
-        })
-        .catch((error) => {
-            onError?.()
-            // eslint-disable-next-line no-console
-            console.error(error)
-        })
+export function launchToOCP(urlSuffix: string, newTab: boolean) {
+    if (newTab) {
+        window.open(`/${urlSuffix}`)
+    } else {
+        location.href = `/${urlSuffix}`
```

Comment:
I believe this will result in an unnecessary page load. It would be smoother to use history.push from react-router-dom. But it looks like we are not currently using this anywhere in the code, so I would probably opt to remove it for now. I also noticed that `launchToOCP` is passed as a property of several components for seemingly no reason. Could we remove these?